cdc/processor/pipeline/system/system.go

```diff
@@ -42,3 +45,21 @@ func (s *System) Start(ctx context.Context) error {
 func (s *System) Stop() error {
 	return s.tableActorSystem.Stop()
 }
+
+func (s *System) Router() *actor.Router {
+	return s.tableActorRouter
+}
+
+func (s *System) System() *actor.System {
+	return s.tableActorSystem
+}
+
+// ActorID returns an ActorID correspond with tableID.
+func ActorID(changefeedID string, tableID model.TableID) actor.ID {
```

Comment:
What about hash collision? Could you generate ID from a global id generator? E.g., an increase-only atomic variable.

Reply:
done